src/main/java/com/notably/commons/core/path/RelativePath.java

```diff
@@ -1,53 +1,112 @@
 package com.notably.commons.core.path;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 import com.notably.commons.core.path.exceptions.InvalidPathException;
 
 /**
  * TODO: Add Javadoc
  */
 public class RelativePath implements Path {
+    public static final String INVALID_RELATIVE_PATH = "Invalid relative path";
+    public static final String VALIDATION_REGEX = "((\\.|\\..|\\p{Alnum}+))+\\/?";
+    private final List<String> paths;
+    private RelativePath(String relativePathString) {
+        this.paths = new ArrayList<>();
+        for (String obj : relativePathString.split("/")) {
+            if (!obj.trim().equals("")) {
+                this.paths.add(obj.trim());
+            }
+        }
+    }
+
     /**
-     * TODO: Add Javadoc
+     * Check if string is a valid relative path.
+     * @param relativePathString String used to check validity.
+     * @return returns true if String is a valid relative Path.
      */
     public static boolean isValidRelativePath(String relativePathString) {
-        // TODO: Add implementation
-        return false;
+        return relativePathString.matches(VALIDATION_REGEX);
     }
 
     /**
-     * TODO: Add Javadoc
+     * Instantiate a RelativePath object if the string is valid.
+     * @param relativePathString used to create RelativePath.
+     * @return the converted RelativePath object.
+     * @throws InvalidPathException if the String is invalid (Not relative path).
      */
     public static RelativePath fromString(String relativePathString) throws InvalidPathException {
-        // TODO: Add implementation
-        return new RelativePath();
+        if (!isValidRelativePath(relativePathString)) {
+            throw new InvalidPathException("Not a relative path");
+        }
+        return new RelativePath(relativePathString);
+
     }
 
     /**
-     * TODO: Add Javadoc
+     * Convert from absolute path to relative path.
+     * @param absolutePath to convert to relative path.
+     * @param currentWorkingPath of the current working directory.
+     * @return the converted relative path.
+     * @throws InvalidPathException
      */
     public static RelativePath fromAbsolutePath(AbsolutePath absolutePath,
             AbsolutePath currentWorkingPath) throws InvalidPathException {
-        // TODO: Add implementation
-        return new RelativePath();
+        return absolutePath.toRelativePath(currentWorkingPath);
```

Comment:
Should we move `AbsolutePath`'s `toRelativePath`'s implementation here instead?

Then, over there in `AbsolutePath`'s `toRelativePath` method we can call `RelativePath.fromAbsolutePath(this, currentWorkingPath)`!

The rationale of moving the implementation there down here is to keep consistent with how `AbsolutePath.fromRelativePath` and `RelativePath`'s `toAbsolutePath` interact!